app/classes/advanced_searches/client_fields.rb

```diff
@@ -13,21 +13,37 @@ def initialize(options = {})
     end
 
     def render
+      common_group          = format_header('common_searches')
       group                 = format_header('basic_fields')
+      care_plan_group       = format_header('care_plan')
       referee_group         = format_header('referee')
       carer_group           = format_header('carer')
       legal_docs            = format_header('legal_documents')
+      case_history_group    = format_header('case_history')
+      family_group          = format_header('family')
+      case_note_group       = format_header('case_note')
+      other_group           = format_header('other')
 
       number_fields         = number_type_list.map { |item| AdvancedSearches::FilterTypes.number_options(item, format_header(item), group) }
+      number_fields         += case_history_number_type_list.map { |item| AdvancedSearches::FilterTypes.number_options(item, format_header(item), case_history_group) }
       text_fields           = text_type_list.map { |item| AdvancedSearches::FilterTypes.text_options(item, format_header(item), group) }
-      text_fields           << referee_text_fields.map { |item| AdvancedSearches::FilterTypes.text_options(item, format_header(item), referee_group) }
-      text_fields           << carer_text_fields.map { |item| AdvancedSearches::FilterTypes.text_options(item, format_header(item), carer_group) }
-      date_picker_fields    = date_type_list.map { |item| AdvancedSearches::FilterTypes.date_picker_options(item, format_header(item), group) }
-      date_picker_fields    += [['no_case_note_date', I18n.t('advanced_search.fields.no_case_note_date')]].map{ |item| AdvancedSearches::CsiFields.date_between_only_options(item[0], item[1], group) }
```

Comment:
@Bunhok Do we have to remove these mappings?

Reply:
Yes, we do.